### PR TITLE
{vis}[GCCcore/13.3.0] FFmpeg v7.0.2, FriBidi v1.0.15, ffnvcodec v12.2.72.0, ...

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.183.01, Python 3.9.18
See https://gist.github.com/SebastianAchilles/77db87d6cc653c19b57704cb64c0b2b5 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/x/x265/x265-3.6-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['663531f341c5389f460d730e62e10a4fcca3428ca2ca109693867bc5fe2e2807']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('CMake', '3.29.3'),
+    ('Yasm', '1.3.0'),
+]
+
+configopts = '-DGIT_ARCHETYPE=1'
```
Comment (smoors) [inline easybuild/easyconfigs/x/x265/x265-3.6-GCCcore-13.3.0.eb:25]: you removed `-DENABLE_PIC=ON` here, is that intentional?

File: easybuild/easyconfigs/l/LAME/LAME-3.100-GCCcore-13.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ('binutils', '2.42'),
+    ('Autotools', '20231222'),
+]
+
+dependencies = [('ncurses', '6.5')]
+
+preconfigopts = "autoconf && "
+
+# configure is broken: add workaround to find libncurses...
+configure_cmd_prefix = "FRONTEND_LDADD='-L${EBROOTNCURSES}/lib' "
```
Comment (smoors) [inline easybuild/easyconfigs/l/LAME/LAME-3.100-GCCcore-13.3.0.eb:29]: nitpicking here, but seems more appropriate to add it to preconfigopts?
```suggestion
preconfigopts += "FRONTEND_LDADD='-L${EBROOTNCURSES}/lib' "
```

File: easybuild/easyconfigs/x/x265/x265-3.6-GCCcore-13.3.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/x/x265/x265-3.6-GCCcore-13.3.0.eb]: Oh, that was unintentional. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/21241/commits/4ee2a2d3c83587cbfe1c4914d465cd5c5ea72a4a

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.183.01, Python 3.9.18
See https://gist.github.com/SebastianAchilles/e597d26b0b565e9bea3d356eec708aa3 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**